packages/react/src/components/Icon/index.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createRef } from 'react';
+import { render } from '@testing-library/react';
+import Icon from './index';
```

Comment:
Very very minor nit.

```suggestion
import Icon from './';
```